composer.json

```
@@ -15,15 +15,15 @@
         "forum": "https://discourse.laminas.dev/"
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
```

```suggestion
        "php": "^7.4 || ~8.0.0 || ~8.1.0",
```
IMO 7.3 support can be droppeed here too as many (most?) other laminas packages already dropped it.